raw_data/preprocessed/africa/zimbabwe_english.txt

```diff
@@ -0,0 +1,486 @@
+
+Emission by motor vehicles and other conveyances
+(68) [1] No owner or operator of a transport conveyance shall operate it in such manner or condition as to cause air pollution in contravention of prescribed emission standards for the class of transport conveyance concerned. [2] Subject to this section, any operator of a transport conveyance who pollutes the environment by contravening the emission standards referred to in subsection [1] commits an offence and shall [a] on conviction for a first offence, be liable to a fine not exceeding level three; [b] on conviction for a second or subsequent offence, be liable to a fine not exceeding level six or in default of payment, to imprisonment for a period not exceeding six months. [3] Any operator of a transport conveyance who admits or is prepared to admit to a police officer or inspector that he is guilty of committing the offence referred to in subsection [2] shall, if he has not been previously convicted of the offence and has not previously been given a warning in terms of this subsection, be given a written warning by the police officer or inspector to undertake, within such period (not less than three months) as may be agreed between him and the police officer or inspector, such work or steps as may be necessary to prevent the contravention by the transport conveyance of the emission standards prescribed for the class of transport conveyance to which it belongs. [4] Where, in respect of an offence, referred to in subsection [2] [a] any person [i] has not complied with a written warning issued in terms of subsection [3]; or [ii] has been previously convicted of the offence referred to in subsection [3]; and [b] the police officer or inspector has reasonable grounds for believing that the magistrates court which will try the person referred to in paragraph [a] for such offence will, on convicting that person of such offence, not impose a sentence of imprisonment or a fine exceeding level three; the person referred to in paragraph [a] may sign and deliver to the police officer or inspector referred to in paragraph [b] a document admitting that he is guilty of the said offence and deposit with such officer or inspector such sum of money as the latter may fix, not exceeding level three, and such person shall thereupon, subject to subsection [5], not be required to appear in court to answer the charge of having committed the said offence. [5] Section 356 of the Criminal Procedure and Evidence Act [Chapter 9:07] shall apply to the procedure to be followed in relation to an admission of guilt made under subsection [4]. [6] Any sum deposited in terms of subsection [4] shall, if deposited with an inspector, form part of the funds of the Agency: Provided that any balance in excess of a sum so deposited which is recovered in terms of section 348 of the Criminal Procedure and Evidence Act [Chapter 9:07] shall not form part of the funds of the Agency.
```

Comment:
Please replace `shall [a]` with `shall: [a]`. Also, please replace `[2] [a] any person [i]` with `[2]: [a] any person: [i]`. Furthermore, should `three; the person referred to` read `three; [c] the person referred to`? Finally, please replace `Agency: Provided` with `Agency, provided`.

Reply:
Regarding 

> Furthermore, should three; the person referred to read three; [c] the person referred to?

I think you are correct.